x/axelarnet/keeper/msg_server.go

```diff
@@ -430,11 +426,16 @@ func toICS20(ctx sdk.Context, k types.BaseKeeper, coin sdk.Coin) sdk.Coin {
 }
 
 // isFromCosmosChain returns true if the asset origins from cosmos chains
-func isFromCosmosChain(ctx sdk.Context, k types.BaseKeeper, coin sdk.Coin) bool {
-	chain, ok := k.GetCosmosChainByAsset(ctx, coin.GetDenom())
+func isFromCosmosChain(ctx sdk.Context, k types.BaseKeeper, n types.Nexus, coin sdk.Coin) bool {
```

Comment:
Axelarnet itself is also a cosmos chain. Maybe rename this function to something like `IBCPathFound`?